GDJS/Runtime/runtimescene.ts

```diff
@@ -744,6 +748,14 @@ namespace gdjs {
             } else {
               this._allInstancesList.push(list[j]);
             }
+            const layerName = list[j].getLayer();
+            const zOrder = list[j].getZOrder();
```

Comment:
Maybe `list[j]` should be set in a local variable because it's used 3 times.

Reply:
Good idea!